project/cmake/modules/FindVAAPI.cmake

```diff
@@ -29,6 +29,10 @@ elseif(VAAPI_INCLUDE_DIR AND EXISTS "${VAAPI_INCLUDE_DIR}/va/va_version.h")
   unset(vaapi_version_str)
 endif()
 
+if(VAAPI_VERSION_STRING VERSION_LESS 0.34)
+message(FATAL_ERROR "VAAPI minimum version 0.34 not found")
+endif()
+
 if(NOT VAAPI_FIND_VERSION)
   set(VAAPI_FIND_VERSION 0.38.0)
```

Comment:
@fetzerch I just tested this. According to you cmake should fail if I set set(VAAPI_FIND_VERSION 0.40.0), because I have 0.39 installed.

But it doesn't fail at all....